apps/cowswap-frontend/src/common/containers/LimitOrdersPromoBanner/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useAtom } from 'jotai'
+import { useCallback } from 'react'
+
+import { useTradeNavigate } from 'modules/trade'
+
+import { Routes } from '../../constants/routes'
+import { LimitOrdersPromoBanner as LimitOrdersPromoBannerPure } from '../../pure/LimitOrdersPromoBanner'
+import { limitOrdersPromoDismissedAtom } from '../../state/limitOrdersPromoAtom'
+
+interface LimitOrdersPromoBannerProps {
+  isLimitOrdersTab?: boolean
+}
+
+export function LimitOrdersPromoBanner({ isLimitOrdersTab = false }: LimitOrdersPromoBannerProps) {
```

Comment:
Why does it call `LimitOrdersPromoBanner` and it is located in `common` directory?